app/models/school.rb

```diff
@@ -32,4 +32,25 @@ def sierra_code
 
     return match.sierra_code
   end
+
+  
+  # Get all active schools
+  def self.active_schools_data
+    active.map do |school|
+      School.name_id(school)
+    end
+  end
+
+
+  # Parse schoool name and id
+  def self.name_id(school)
```

Comment:
Hi Deepika, any reason that this and the next should be class methods rather than instance methods?

Reply:
(I realise you inherited a lot of this code, so wasn't your decision initially)